raster-foundry/raster-foundry
# Use correct cloud coverage field for Sentinel-2

## Overview

This PR: 
- For import, use correct cloud coverage field in sentinel-2 metadata
- In DB, update sentinel-2 `cloud_cover` column to use the `cloudyPixelPercentage` field in `scene_metadata` column


### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

## Testing Instructions

 * Run migration
 * Check in the DB if sentinel-2 scenes have the same `cloud_cover` values as the `cloudyPixelPercentage` filed in `scene_metadata`. OR bring up frontend and browse sentinel-2 scene details to check if the two fields match.
 * Run sentinel-2 scene import in batch and check if new records have the correct cloud coverage.

Closes #3603
